### DISCUSSION TITLE
The difference of Cmax in table and graph of pantoprazole

### DISCUSSION THREAD

**Discussion by Vviin** [Open-Systems-Pharmacology/Forum#1493] [Questions & Problems]
I developed the PBPK model of pantoprazole based on the graph. and the pantoprazole model perfectly fit the graph but there is big gap between graph and table of cmax. In the graph Cmax value is about 1400-1500ng/ml but in the table the value is 2250nm/ml...
What should I do? :(

**Comment by StephanSchaller** [comment]
Hi @Vviin, please provide more details. Which graph and Table do you mean?...Do you have FIgures/Screenshots? It is hard to answer your question without further info.

**Reply by Vviin** [reply]
![1](https://user-images.githubusercontent.com/129701811/229470388-4db7baf0-5c8a-4b25-a09c-ccac1aaf08f6.png)

![2](https://user-images.githubusercontent.com/129701811/229470427-e2156145-4725-40d6-ac1b-8580c0ab67ac.png)

**Reply by Vviin** [reply]
in first figure, individual PK value of Cmax is 1601ng/ml and in second figure, aggregated PK value is 1582ng/ml

**Reply by Vviin** [reply]
but , in observed data.. individual PK value is 2400ng/ml which is greater than 1600ng/ml cmax of aggregated PK value..
![image](https://user-images.githubusercontent.com/129701811/229471238-5b83334b-e482-49dc-9fd0-141d005db947.png)

**Reply by Yuri05** [reply]
* In the 2<sup>nd</sup> tab ("**Aggregated PK Values**") you see the values calculated on the displayed aggregated curves. Thus here you see the CMax of the plotted curve (it's not clear from your plot, but I assume you are ploting the **median** curve), and the value 1582.5 [ng/ml] pretty much matches the CMax of the plotted curve.
* In the 1<sup>st</sup> tab ("**Individual PK Values**") you see the median values from the individual curves. 
E.g. here you see  `CMax=Median{Individual CMax}` - which in the most cases would be different from the CMax of the median curve.

**Reply by Vviin** [reply]
I think there is too much difference between the observed data's gap (aggregate PK value & median Cmax of individuals) and simulated data's gap so.. I want to figure out it or want to fit that gap but so hard
